packages/create-app/template-svelte-ts/package.json

```diff
@@ -16,5 +16,6 @@
     "tslib": "^2.2.0",
     "typescript": "^4.3.2",
     "vite": "^2.3.7"
-  }
-}
```

Comment:
Would you add the new line here and in the other template so the diff is more clear? @Shinigami92 are we having these issues because eslint is not applied to create-app templates?

Reply:
Actually, this looks fine, comparing to others templates package.json